.github/workflows/docs.yml

```
@@ -26,4 +26,10 @@ jobs:
         pip install .
     - name: Build docs
       run: |
-        cd doc && make install
+        cd doc && make html
+    - name: Deploy Docs
+      uses: peaceiris/actions-gh-pages@v3
```

Only deploy on master branch.

```suggestion
      uses: peaceiris/actions-gh-pages@v3
      if: github.ref == 'refs/heads/master'
```